### PR TITLE
Change "L/t" to "mb/t" for consistency

### LINKED ISSUE #1378: Change "L/t" to "mb/t" to reduce confusion
### Cross-mod Integration

_No response_

### Feature Description

The mod shouldn't specialize itself by using "Liter/tick" (from here on out as "L/t"). It's highly confusing and causes more issues than necessary.

Change the nomen-clature from "L/t" to "mb/t" to keep in line with the last 10 years of modded nomenclature. Or at-least provide a setting to change it from "L/t" to "mb/t".

This is also highly inconsistent as "Liter" in the realworld is 1000 milliliter (in-game as millibuckets as 1 Bucket = 1 L).
The tanks also use Buckets (or millibuckets in recipes). Thus being highly confusing for no other reason than the sake of it being that way.

This issue is also very easily fixable and has no backlash and it would make it easier for newer players to gregtech to understand the various units of measurement in-game.

### PR DESCRIPTION
**Pull request by 3x1t-5tyl3**
## What
#1378 

## Implementation Details
None. It only changes certain comments and lang files as-well as top and jade compat to show mb/t instead of L/t

## Outcome
See above.

## Additional Information
None.

## Potential Compatibility Issues
None.

### CODE REVIEW

**Comment by screret** [comment]
It should be "mB", not "mb"

**Comment by 3x1t-5tyl3** [comment]
> It should be "mB", not "mb"

Whoops. Copy paste fail. I'll fix that.
EDIT: Fixed it lol. Fixes my own typos.

**Review by YoungOnionMC — APPROVED** [review]

**Comment by screret** [comment]
please run the `spotlessApply` task in your IDE.

File: src/main/java/com/gregtechceu/gtceu/data/lang/LangHandler.java [hunk 1]
```diff
@@ -796,6 +796,7 @@ public static void init(RegistrateLangProvider provider) {
         provider.add("gtceu.universal.enabled", "Multiblock Sharing §aEnabled");
         provider.add("gtceu.universal.liters", "%s L");
         provider.add("gtceu.universal.kiloliters", "%s kL");
+        // Not sure if the Liter is still wanted? I'll keep these as Liter given they're named properly
```
Comment (screret) [inline src/main/java/com/gregtechceu/gtceu/data/lang/LangHandler.java]: do change them.

File: src/generated/resources/assets/gtceu/lang/en_ud.json [hunk 2]
```diff
@@ -2824,7 +2824,7 @@
   "gtceu.machine.hpca.component_type.computation_cooling": "buıןooƆ %dɟ§ :oʇ dn sǝɹınbǝᴚɔ§",
   "gtceu.machine.hpca.component_type.computation_cwut": "ʇ/∩MƆ %dɟ§ :uoıʇɐʇndɯoƆ6§",
   "gtceu.machine.hpca.component_type.cooler_active": "ǝʌıʇɔⱯɟ§ :ǝdʎ⟘ ɹǝןooƆq§",
-  "gtceu.machine.hpca.component_type.cooler_active_coolant": "%s ʇ/Ꞁ %dɟ§ :oʇ dn sǝɹınbǝᴚɔ§",
+  "gtceu.machine.hpca.component_type.cooler_active_coolant": "%s ʇ/qɯ %dɟ§ :oʇ dn sǝɹınbǝᴚɔ§",
```
Comment (screret) [inline src/generated/resources/assets/gtceu/lang/en_ud.json]: rerun datagen please.

**Review by screret — CHANGES_REQUESTED** [review]

**Review by screret — APPROVED** [review]

**Review by screret — APPROVED** [review]

**Pull request merged**